markdown/docs/user-guide.md

```diff
@@ -28,370 +37,455 @@ You can start a FastBoot server on your development machine by running:
 ember fastboot
 ```
 
-This starts a FastBoot server listening on port 3000. You can verify
-it's working by curling from `localhost`:
+This starts a FastBoot server listening on port 3000\.
+You can verify it's working by curling from `localhost`:
 
 ```sh
 curl http://localhost:3000
 ```
 
+You should see your index route fully rendered (as opposed to the skeletal `index.html` page Ember usually sends).
```

Comment:
I think this sentence assumes some knowledge of Ember that new users may not have. It might be better to say something like:

> You should see the content of your application's index template rendered in the output, instead of the empty HTML most client-side rendered apps show.